karapace/kafka_rest_apis/__init__.py

```diff
@@ -653,9 +653,9 @@ def init_admin_client(self):
                 break
             except:  # pylint: disable=bare-except
                 if retry:
-                    log.exception("Unable to start admin client, retrying")
+                    log.warning("Unable to start admin client, retrying")
                 else:
-                    log.exception("Giving up after failing to start admin client")
+                    log.info("Giving up after failing to start admin client")
```

Comment:
If retry is true, log level should be lower (info) than when retry is false (last try failing). Switch info and warning between each other here?